dss/storage/validator.py

```diff
@@ -32,34 +100,28 @@ def remove_json_fields(json_data: dict, path: List[str], fields: List[str]):
         current.pop(field)
 
 
-DSS_Draft4Validator = validators.create(
-    meta_schema=_utils.load_schema("draft4"),
-    validators={
-        u"$ref": _validators.ref,
-        u"additionalProperties": _validators.additionalProperties,
-        u"properties": _validators.properties_draft4,
-        u"required": _validators.required_draft4,
-    },
-    version="draft4",
-)
-
-resolver = validators.RefResolver(referrer='', base_uri='')
-
-
 def scrub_index_data(index_data: dict, bundle_id: str, logger: logging.Logger) -> list:
+    handler = S3UrlCache(logger)
```

Comment:
Rename to `cache`. Add `def request_json(…): return json.loads(cache.resolve())`

Reply:
how about this
```python
    cache = S3UrlCache(logger)
    request_json = lambda url: json.loads(cache.resolve(url).decode("utf-8"))
```